gui/src/components/SettingsNetwork.tsx

```diff
@@ -187,6 +199,14 @@ export function SettingsNetwork() {
           Add network
         </Button>
       </Stack>
+      <Button
+        variant="outlined"
+        color="warning"
+        size="medium"
+        onClick={onReset}
+      >
+        Reset Networks
+      </Button>
```

Comment:
What's the need for this?
also, sounds rather dangerous to have a 1-click reset of important settings

Reply:
It's for when I mess up the networks and want the defaults back. I can add a confirmation modal maybe?

Reply:
I think that would help, yes